src/utils/server/ipfs/infuraScheduler.ts

```diff
@@ -0,0 +1,18 @@
+import Bottleneck from 'bottleneck'
+
+// Back off for 3 secs - pinata rate limits for 180/min
```

Comment:
Change these settings to match infura IPFS backoff

Reply:
ah good catch, done!